pypeit/spectrographs/keck_kcwi.py

```diff
@@ -463,3 +512,48 @@ def bpm(self, filename, det, shape=None, msbias=None):
 
         return bpm_img
 
+    def set_wcs(self, hdr):
```

Comment:
Have you played with the `astropy.wcs` package much?  I've only used it to read and construct WCS objects from headers, but I'm wondering if there are some utilities that will make this setup easier, and make sure that it conforms to an expected standard.

Reply:
I haven't no, but I definitely intend to use it. This is (almost) a straight copy of the KDERP IDL pipeline... I ambitiously made a start on it, but I think I'm a way off that yet!